Extensions/TeamCity/Src/vss-extension.json

```diff
@@ -73,7 +73,7 @@
           },
           {
             "name": "Builds",
-            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds",
+            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds/?locator=branch:default:any",
```

Comment:
@omeshp @GitHubSriramB This approach obviously doesn't scale very well with a lot of branches or builds. Better would be a solution where the branches can be defined while creating the artifact link, but I don't know if this is posisble from TFS side, and also by the TeamCity REST API (would require wildcards for branches).